test/end-to-end/README.md

```diff
@@ -105,18 +105,18 @@ describe('View Recipe Page', function () {                // Which page does thi
     after((done) => {                                     // Scenario de-provision, like apply a snapshot
     });
 
-    context('Menu Nav Bar Check', () => {                 // Element or elements to be test
-      it('should show a recipe name', (done) => {         // Expected result description
+    context('Menu Nav Bar Check #Acceptance', () => {     // Test content and test type
+      it('should show a recipe name @Welder-Web-40003', (done) => {    // Case description and case ID
```

Comment:
What are these numbers? I don't like having this defined by hand b/c it will get out of sync pretty quickly. What is the purpose of having the numbers ?